### PR TITLE
Add digest images surrogate keys

### PR DESCRIPTION
**Pull request by giorgiosironi**
Follow-up to https://github.com/elifesciences/issues/issues/4577 which required blood and tears to fix.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Applied to `iiif--continuumtest`, purged all, but https://continuumtest--cdn-iiif.elifesciences.org/digests%3A33947%2Fdigest-33947.jpg/full/full/0/default.jpg shows no difference.

**Comment by giorgiosironi** [comment]
Actually surrogate keys don't work also on
https://iiif.elifesciences.org/lax:39689%2Felife-39689-fig1-v2.tif/full/1234,/0/default.jpg
which is an existing article image?

**Comment by giorgiosironi** [comment]
Maybe they are magically stripped?

**Comment by giorgiosironi** [comment]
Managed to see them on an article figure with the debug header, but not working on digests yet:
```
$ curl -v -H "Fastly-Debug: 1" -o /dev/null https://continuumtest--cdn-iiif.elifesciences.org/digests/33947%2Fdigest-33947.jpg/full/full/0/default.jpg
```

File: projects/elife.yaml [hunk 1]
```diff
@@ -1695,6 +1695,24 @@ iiif:
                         encoded-urls:
                             path: /lax/26866%2Felife-26866-fig3-video1.jpg
                             expected: "article/26866 article/26866/videos"
+                digest-id:
+                    url: "^/digests/([0-9]+).*$"
+                    # doesn't work:
+                    #url: "^/digests/([0-9]+)(?:/|%2F)(?:.+)$"
```
[the remark this answers is not in the document]
Comment (giorgiosironi) [reply projects/elife.yaml]: Trying it on `continuumtest`

[the remark this answers is not in the document]
Comment (giorgiosironi) [reply projects/elife.yaml]: Doesn't seem to work on `curl -v -H "Fastly-Debug: 1" -o /dev/null https://continuumtest--cdn-iiif.elifesciences.org/digests/33947%2Fdigest-33947.jpg/ful
l/full/0/default.jpg` after applying and doing a purge all.

[the remark this answers is not in the document]
Comment (thewilkybarkid) [reply projects/elife.yaml]: Was the space meant?

[the remark this answers is not in the document]
Comment (giorgiosironi) [reply projects/elife.yaml]: Bad copy paste, corrected.

**Pull request merged**